web/packages/teleterm/src/ui/TopBar/Identity/Identity.story.tsx

```diff
@@ -33,7 +33,7 @@ export default {
 
 const makeTitle = (userWithClusterName: string) => userWithClusterName;
 const profileStatusError =
-  'No YubiKey device connected with serial number 14358031';
+  'No YubiKey device connected with serial number 14358031. Connect the device and try again.';
```

Comment:
This seems unrelated. Are you sure it should also be here?

Reply:
I made the text longer in the story to see what it looks like when it is longer than the container.